pkg/controller/controller_binding.go

```diff
@@ -166,7 +166,7 @@ func (c *controller) reconcileBinding(binding *v1alpha1.Binding) error {
 
 		var parameters map[string]interface{}
 		if binding.Spec.Parameters != nil {
-			parameters, err = unmarshalParameters(binding.Spec.Parameters.Raw)
+			parameters, err = unmarshalRawParameters(binding.Spec.Parameters.Raw)
```

Comment:
Why doesn't Binding use the same Parameters objects as Instance?  I see it defined as `Parameters *runtime.RawExtension` in the BindingSpec.